account_banking_payment_export/models/payment_mode.py

```diff
@@ -104,3 +104,7 @@ def type_on_change(self):
                 aj_ids = ajo.search([
                     ('type', 'in', ('sale_refund', 'sale'))]).ids
             self.default_journal_ids = [(6, 0, aj_ids)]
+
+    bank_id = fields.Many2one(required=False)
+    check_required = fields.Boolean(default=True, string="Bank account required",
```

Comment:
You should include the check_required at payment type level, not at payment mode. This way, "Manual Direct Debit" and "Manual Bank Payment" have this check unmarked, and SEPA payment types have it checked. The default value for the field should be True to preseve the same behaviour.